tools/__tasks__/compile/javascript/index.js

```diff
@@ -1,12 +1,12 @@
-module.exports = [{
-    title: 'Compile JS',
+module.exports = {
+    description: 'Compile JS',
     task: [
         require('./clean'),
-        require('../inline-svgs/compile'),
+        'compile/inline-svgs',
```

Comment:
yeah it's a fair point. it's true that my focus so far has been getting the tasks out of grunt in a (hopefully) reasonably understandable way. i think you may be correct there though...